RELEASING.md

```diff
@@ -0,0 +1,59 @@
+Releasing
+=========
+
+Steps to follow to release a new version of pyIATI.
+
+1. In the `dev` branch.
+    1. Ensure license information for any new dependencies has been added to `LICENSES-3RD-PARTY.txt`.
+    2. Read the README to ensure it is up-to-date (if it isn't, fix it!).
+    3. Ensure that all relevant Pull Requests have been merged into `dev`.
+    4. Ensure the `Unreleased` section in `CHANGELOG.md` covers all changes between `  dev` and `master`.
+    5. Ensure all lines under `Unreleased` in `CHANGELOG.md` include a reference to a Pull Request or Issue.
+    6. Bump the version.
```

Comment:
Suggest `Bump the version.` -> `Bump the version, as per [SemVer](http://semver.org/).`

Reply:
In fact, how about

> 6. Prepare the release. You will need a new pull request for these steps:
1. Bump the version, as per [SemVer](http://semver.org/). Run one of the following as appropriate:
    1. If the changes are bug fixes only: `bumpversion patch`
    2. If new features are added: `bumpversion minor`
    3. There **MUST NOT** be a `major` version increase (to `1.0.0`) until a suitable amount of functionality is stable and tested.
2. Move content under `Unreleased` in `CHANGELOG.md` to a new section, with a heading in the format: `[version-number] - yyyy-mm-dd`.
3. Ensure a blank `Unreleased` section is left in `CHANGELOG.md` (see later in doc for template).

Reply:
Seems sensible - `make docs` would also fall under this.